src/dodal/devices/turbo_slit.py

```diff
@@ -1,8 +1,8 @@
-from ophyd_async.core import Device
+from ophyd_async.core import Device, StandardReadable
 from ophyd_async.epics.motor import Motor
 
 
-class TurboSlit(Device):
+class TurboSlit(StandardReadable, Device):
```

Comment:
```suggestion
class TurboSlit(StandardReadable):
```

Should: StandardReadable already extends Device